### PR TITLE
Move `examples/grpc-service` to `examples/grpc`

### PR DESCRIPTION
**Pull request by trustin**
.. and renamed the project from `grpc-service` to `grpc-example`.

### CODE REVIEW

**Comment by trustin** [comment]
I'm not 100% sure, but my guess is that `.gitkeep` files are to signify that a user can put some files there. I'm not sure if it's a good convention or not though. I'm OK with removing them.

File: settings.gradle [hunk 1]
```diff
@@ -50,7 +50,8 @@ includeWithFlags ':site'
 
 // Examples
 includeWithFlags ':examples:annotated-http-service', 'java'
-includeWithFlags ':examples:grpc-service',           'java'
+includeWithFlags ':examples:grpc-example',           'java'
```
Comment (ikhoon) [inline settings.gradle]: nit: How about `grpc-minimal`? And we could adapt #2310 as `grpc-reactive`?

I'm not sure this suggestion is better than `grpc-example`. 🙃 You can feel free to ignore this.

**Comment by anuraaga** [comment]
I'd lean towards removing them or adding a text string inside with the motivation. But off the top of my head, I'm not sure if Java ignores dot files in resources - if not I guess it's not great to add dummy files to the result of the project.

**Comment by trustin** [comment]
I don't think Java or Gradle ignores dot files. Do they?

File: settings.gradle (hunk 1 above)
Comment (trustin) [reply settings.gradle]: I don't know either. I guess it doesn't matter much anyway though. Let me leave it as it is :wink:

**Comment by anuraaga** [comment]
Yeah I guess they do - so to keep output clean, maybe better to not have them. But not a huge deal either way

**Pull request merged**